app/lib/publisher/backend.dart

```diff
@@ -91,6 +91,23 @@ class PublisherBackend {
     return _asPublisherInfo(p);
   }
 
+  /// List the members of a publishers
+  Future<api.PublisherMembers> listPublisherMembers(String publisherId) async {
```

Comment:
The endpoint should not do caching, because we need to authenticate and then authorize the request, and doing that with the code generator would add too much complexity.

Reply:
(adding TODO)

Reply:
Ah, this is still ambiguous: the generated HTTP endpoint should not do the caching, as the implementation does the authorization. We could do memcache-based caching in the implementation method, I'm adding a TODO for it.